search.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+//display the list results
+var appendArticleAttributes = function(article) {
+
+  var title = document.createElement('div');
+  var section = document.createElement('div');
+  var byline = document.createElement('div');
+  var link = document.createElement('a');
+  var li = document.createElement('li');
+
+	title.textContent = "Title: " + article.title;
+  section.textContent = "Section: " + article.section;
+  byline.textContent = "Byline: " + article.byline;
+  link.textContent = "Link";
+  link.href = article.url;
+
+  li.appendChild(title);
+  li.appendChild(section);
+  li.appendChild(byline);
+  li.appendChild(link);
+
+  list.appendChild(li);
```

Comment:
I don't see where `list` is defined and at first I was confused how your solution was able to work. After some exploration I discovered that elements with an id attribute can automatically be referenced in js by their id. I'm really scratching my head on this one, as I've never seen a reference to this feature before, but I've it works in the latest Chrome, Safari, Firefox, and Opera, so definitely not a fluke. Was this your intention - in which case I'd be confused why you'd define `searchButton` on line 60 and use `getElementById` on line 64 - or was this a happy accident - which would explain why `articleList` was defined on line 61?

Reply:
Happy accident! Wow.  Almost wish that didn’t work because it prevented me from catching that typo.  I feel like “use strict” shouldn’t allow this to work because it absolutely prevented me from catching a bug.  Something I’ll look into further as I’ve never seen this before either.